EdgeGridAuth/EdgeGridAuth.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Akamai.EdgeGrid.Auth</RootNamespace>
     <AssemblyName>Akamai.EdgeGrid.Auth</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
What impact does this have on users? Should this be considered a BC break?

Reply:
https://stackoverflow.com/questions/28286086/default-securityprotocol-in-net-4-5

Reply:
@justinrummel OK, so it's required for TLS 1.2, which means we can't avoid it, and users will be _required_ to update if necessary.